### ISSUE TITLE
Peak loss in log-scale contour plots

### ISSUE THREAD

**Issue by annliirosales** [jamesETsmith/msanalysis#2]
Low intensity peaks don't show up in the current log scale contour plots (example 4). One resolution would be to remove the intense bands from precursor that are present throughout the temperature ramp. This requires input of m/z ranges of the intense bands that the users wish to exclude in the codes. Hopefully this will allow the low intensity peaks to show.

**Comment by annliirosales** [comment]
Hi @jamesETsmith , I was able to get some low intensity peaks to show up in the contour by normalizing the intensity to a large number (> 1000). However, the background looks very patchy. Perhaps some noise is getting picked up? See below for examples. Maybe removing high intensity precursor bands will remedy this. Let me know if you'd like the data file that I used to generate the plots. Thank you!
![max:100](https://user-images.githubusercontent.com/56844738/76043546-19cc6900-5f15-11ea-8227-0b897c1f3728.png)
![max:1500](https://user-images.githubusercontent.com/56844738/76043547-1afd9600-5f15-11ea-93ed-8a0e41d4a6a9.png)
